templates/zone_record.erb

```diff
@@ -1,8 +1,14 @@
 <%
-delimiter = %w[TXT SPF].include?(@record.upcase) ? '"' : ''
 data_a = @data.is_a?(String) ? @data.lines.to_a : @data
 
 data_a.each do |data_item|
+if %w[TXT SPF].include?(@record.upcase)
+  # for TXT and SPF records, transform the data item into a valid text record value:
+  # use String.unpack to split the text value into 255 character chunks;
+  # then use Array.collect to turn each chunk into a valid quoted character string;
+  # finally join the quoted character strings together, separated by a space.
```

Comment:
You did a great job documenting why we do this in the issue, can you put the abridged version here? (at least link to the RFC)